flake.nix

```diff
@@ -64,7 +64,7 @@
           name = "SOUFFLE-HASKELL";
           packages = with haskellPackages; [
             cabal-install
-            ghc
+            haskellPackages.ghc
```

Comment:
Seems redundant and can be kept same as before? There's a `with haskellPackages` on line 65?

Reply:
Sure, will fix this soon